db/dcrpg/pgblockchain.go

```diff
@@ -485,10 +445,10 @@ type DBInfo struct {
 // needed if database upgrades are required.
 func NewChainDB(dbi *DBInfo, params *chaincfg.Params, stakeDB *stakedb.StakeDatabase,
 	devPrefetch, hidePGConfig bool, addrCacheCap int, mp rpcutils.MempoolAddressChecker,
-	parser ProposalsFetcher, bg BlockGetter) (*ChainDB, error) {
+	parser ProposalsFetcher, client *rpcclient.Client, shutdown func()) (*ChainDB, error) {
```

Comment:
Note to self: major version bump up to db/dcrpg/v5.  This plus the large DB upgrade is why I'm thinking 5.2 perhaps should become 6.0, otherwise this PR can go on hold until 5.2 commits are all done.